main/strlist.c

```diff
@@ -216,7 +216,21 @@ static bool fileNameMatched (
 		const vString* const vpattern, const char* const fileName)
 {
 	const char* const pattern = vStringValue (vpattern);
+
+#ifdef CASE_INSENSITIVE_FILENAMES
+	{
+		bool r;
+
+		char* const p = newUpperString (pattern);
+		char* const f = newUpperString (fileName);
+		r =  (bool) (fnmatch (p, f, 0) == 0);
```

Comment:
No big deal, but this `(bool)` cast (and the existing one below) isn't needed. Also the `r` variable could be initialized directly like `bool r = (fnmatch (p, f, 0) == 0);` instead of doing it in two steps.